### DISCUSSION TITLE
Help with one axis chart and callouts in JavaFX

### DISCUSSION THREAD

**Discussion by ad5xj** [jfree/jfreechart#353] [Q&A]
I am creating a chart that displays occurrences of data, each of which I wish to mark with a callout having some detail for each point.

A small example would be a chart of limited-range frequencies, with data occurrences charted at points along the same vertical axis, each point marked by a callout providing detail for each.

I understand that this is probably an unconventional way to display data since only the vertical axis is needed here and all data points are on the same vertical axis.

Problem #1 how do I create the chart with only the vertical axis scale?

Problem #2 how do I create the callout for each charted point?

Problem #3 is there a way to associate the charted point with the callout detail in such a way that I can connect them with a drawn line (per the conventional callout display)?

Having asked this question on stackoverflow.com, I was directed here by one mentor. 
This is my first foray into charting graphics with JavaFX and jFreeChartFX so this may be a newbie question. Any help would be appreciated.

**Comment by trashgod** [comment]
As suggested [here](https://stackoverflow.com/q/75802228/230513), you might want to look at [`org.jfree.chart.annotations`](https://www.jfree.org/jfreechart/javadoc/org/jfree/chart/annotations/package-summary.html) and try a few examples; also look a tooltips and axis visibility. Most [examples](https://stackoverflow.com/questions/tagged/jfreechart) can be easily adapted to JFreeChart-FX.

**Reply by ad5xj** [reply]
This suggestion does not get me past two of the three problems. I don't think annotations provide the whole answer. The examples were available before I posted this message. There were no clear examples that provided any ideas for me.

Thank you for trying.

**Reply by trashgod** [reply]
In addition to tooltips and axis visibility, look at `org.jfree.chart.labels`.

**Comment by ad5xj** [comment]
This crude diagram I created manually, is what I wish to achieve. If there are helpful hints as to how this can be done, I am very appreciative. 
![Diagram1](https://user-images.githubusercontent.com/5191354/227788300-cf8a718f-1273-4df4-970f-6312d588c670.png)

**Reply by trashgod** [reply]
These look like pointer annotations. Browse the [sample demo](https://www.jfree.org/jfreechart/samples.html) and look [here](https://stackoverflow.com/questions/tagged/jfreechart) for specific examples.
